jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -703,6 +703,47 @@ ecma_op_object_get (ecma_object_t *object_p, /**< the object */
   return ecma_make_simple_value (ECMA_SIMPLE_VALUE_UNDEFINED);
 } /* ecma_op_object_get */
 
+/**
+ * [[Get]] ecma object's operation using magic string name
+ *
+ * See also:
+ *          ECMA-262 v5, 8.6.2; ECMA-262 v5, Table 8
+ *
+ * @return ecma value
```

Comment:
Please add more details here. For example it when it will return undefined and when an object value.